docs/TimelineGuide.md

```diff
@@ -5,7 +5,7 @@ primarily for cactbot.
 
 ![import screenshot](images/timelineguide_timeline.png)
 
-cactbot uses the [raidboss module](https://github.com/quisquous/cactbot#raidboss-module)
+Cactbot uses the [raidboss module](https://github.com/quisquous/cactbot#raidboss-module)
```

Comment:
This is a weird request, but can you leave this as lowercase everywhere to be consistent with everything else?